explainer.md

```diff
@@ -0,0 +1,92 @@
+
+
+## Methods
+
+[takePhoto()](https://w3c.github.io/mediacapture-image/#dom-imagecapture-takephoto) returns a captured image encoded in the form of a [Blob](https://www.w3.org/TR/FileAPI/#dfn-Blob), whereas [grabFrame()](https://w3c.github.io/mediacapture-image/#dom-imagecapture-grabframe) returns a snapshot of the track video feed in the form of a non-encoded [ImageBitmap](https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#imagebitmap). takePhoto() produces the result of a single photographic exposure using the video capture device sourcing the track and including any PhotoSettings configured, returning an encoded image in the form of a Blob if successful. grabFrame() takes a snapshot of the live video being held in track, returning an ImageBitmap if successful. Ideally takePhoto could produce the highest quality image the platform can allow for a still photograph.
+
+## API example
+
+```js
+var imageCapture;
+
+  navigator.mediaDevices.getUserMedia({video: true})
+    .then(gotMedia)
+    .catch(err => console.error('getUserMedia() failed: ', err));
+
+  function gotMedia(mediastream) {
+    const video = document.querySelector('video');
+    video.srcObject = mediastream;
+
+    const track = mediastream.getVideoTracks()[0];
+    imageCapture = new ImageCapture(track);
+
+    const capabilities = track.getCapabilities()
+    // Check whether focus distance is supported or not.
+    if (!capabilities.focusDistance) {
+      return;
+    }
+
+    // Map focus distance to a slider element.
+    const input = document.querySelector('input[type="range"]');
+    input.min = capabilities.focusDistance.min;
+    input.max = capabilities.focusDistance.max;
+    input.step = capabilities.focusDistance.step;
+    input.value = track.getSettings().focusDistance;
+
+    input.oninput = function(event) {
+      track.applyConstraints({
+        advanced : [{focusMode: "manual", focusDistance: event.target.value}]
+      });
+    }
+    input.hidden = false;
+  }
+
+  function takePhoto() {
+    imageCapture.takePhoto()
+      .then(blob => {
+        console.log('Photo taken: ' + blob.type + ', ' + blob.size + 'B');
+
+        const image = document.querySelector('img');
+        image.src = URL.createObjectURL(blob);
+      })
+      .catch(err => console.error('takePhoto() failed: ', err));
+  }
+```
+
+## Capabilities
+
+This specification allows web developers to get and set various properties in a modern webcam like `whiteBalanceMode`, `exposureMode`, `focusMode`, `exposureMode`, `pointsOfInterest`, `exposureCompensation`, `exposureTime`, `colorTemperature`, `iso`, `brightness`, `contrast`, `pan`, `saturation`, `sharpness`, `focalDistance`, `tilt`, `zoom` and  `torch` if there is support in the target platform.
```

Comment:
Maybe read and constrain instead of get and set?
```suggestion
This specification allows web developers to read and constrain various properties in a modern webcam like `whiteBalanceMode`, `exposureMode`, `focusMode`, `exposureMode`, `pointsOfInterest`, `exposureCompensation`, `exposureTime`, `colorTemperature`, `iso`, `brightness`, `contrast`, `pan`, `saturation`, `sharpness`, `focalDistance`, `tilt`, `zoom` and  `torch` if there is support in the target platform.
```